apps/sms/views/conversation/style/message.css

```diff
@@ -49,6 +49,7 @@
 
 .message p.message-content-body {
   white-space: pre-wrap;
+  font-family: "Firefox Emoji", sans-serif;
```

Comment:
Sorry, was too busy today :/ Can you , tell me how I can test this PR? I think I've tried to install Emoji keyboard from marketplace, but I was able to see these Emoji without this PR as well, so I'm not sure what should be the difference? I admit I've never used smth like this :p

Also I think it would be better to introduce something like ```.support-emoji``` class e.g. in sms/views/shared/style/common.css and add it everywhere we need it in markup. What do you think?